services/api/src/game/__tests__/game.service.test.ts

```diff
@@ -12,6 +22,18 @@ describe('game.service', () => {
   });
   it('should return empty list when all players are done', async () => {
     const newState = gameService.figureMoved(null);
-    expect(newState).toMatchSnapshot(newState);
+    clearBoard();
+    expect(newState).toMatchObject([]);
+  });
+  it('should clear the board', function () {
+    const emptyArr: PlaceFigureMessage[] = [];
+    turns.push(newTurn);
+    clearBoard();
+    expect(emptyArr).toMatchSnapshot(turns);
```

Comment:
The assertion should be the other way arround. `expect(testedValue).toEqual(expectedValue)` in this case, the tested value is `turns`, not `emptyArr`